config/environments/development.rb

```diff
@@ -3,6 +3,12 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  # Do not dump schema automatically after migrations
+  config.active_record.dump_schema_after_migration = false
```

Comment:
This will break our local development flows for this project since we need to dump the schema when adding migrations. Could you instead run the container in production mode (the default for the `Dockerfile`) which would avoid dumping the schema?

Reply:
I did not want to force things into production mode just to keep everything safe in development mode. See my new diffs which makes things configurable and works well.